### ISSUE TITLE
In case of multiple tables, with thead repeat per page is required, and if first row of second table has longer content then that row is displaying in another page and header is repeating two times

### ISSUE THREAD

**Issue by vk-sourcefuse** [mikehaertl/phpwkhtmltopdf#288]
Generate pdf with 0.12.5
[test multiple tables.pdf](https://github.com/mikehaertl/phpwkhtmltopdf/files/2259525/test.multiple.tables.pdf)
Please refer attached pdf
In case of multiple tables, with header-repeat-per-page, if first row of second table has longer content then it is displaying in another page and header is repeating two times
HTML : 
[test.html.zip](https://github.com/mikehaertl/phpwkhtmltopdf/files/2259527/test.html.zip)

So basically header should not repeat two times in above case for table-2
: either remove first header
: or break that 1st row in two parts, half on first page and half in other page.

tried 
 tr {
      page-break-inside:unset;
 }
too,
That first row overlaps with thead in v 0.12.4, but in 0.12.5 it is more worst.

**Comment by mikehaertl** [comment]
Our library only wraps wkthmltopdf in a PHP class. We can not fix things that are broken in wkthmltopdf itself. You should ask for support over at their [project site](https://github.com/wkhtmltopdf/wkhtmltopdf).

**Comment by vk-sourcefuse** [comment]
Oh, okay, thanks, 
your library is awesome, 
but your library is not working with wkhtmltopdf v0.12.5 
I want to put binary in of v0.12.5 in my project and give path of that binary to execute, as we can do in 0.12.4
like:
$this->pdfObj->binary = /opt/lampp/htdocs/myproject/lib/wkhtmltox/bin/wkhtmltopdf

**Comment by mikehaertl** [comment]
What do you mean by "not working"? You can still configure the path as usual. There's nothing version specific in our library. It should work with any version of wkthmltopdf.

**Comment by vk-sourcefuse** [comment]
wkhtmltopdf command is working fine in terminal. 
And result of 
"which wkhtmltopdf"
is 
"/usr/local/bin/wkhtmltopdf" 

And 
wkhtmltopdf -V
results:
wkhtmltopdf 0.12.5 (with patched qt)

Please see this screenshot:
![screenshot from 2018-08-04 20-13-59](https://user-images.githubusercontent.com/30282624/43677644-f4f96d40-9822-11e8-89ed-6f8b11a861a7.png)

But with this version, (all my code is same), it throws this error : 
`

  | wkhtmltopdf: /opt/lampp/lib/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by wkhtmltopdf)
-- | --
  | wkhtmltopdf: /opt/lampp/lib/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by wkhtmltopdf)
  | wkhtmltopdf: /opt/lampp/lib/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by wkhtmltopdf)
  | wkhtmltopdf: /opt/lampp/lib/libz.so.1: version `ZLIB_1.2.9' not found (required by /usr/lib/x86_64-linux-gnu/libpng16.so.16)

`
If I manually set path of wkhtmltopdf, 
$this->pdfObj = new Pdf($params);
        $this->pdfObj->binary = "/usr/local/bin/wkhtmltopdf";
then it throws following error : 

`

  | /usr/local/bin/wkhtmltopdf: /opt/lampp/lib/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by /usr/local/bin/wkhtmltopdf)
-- | --
  | /usr/local/bin/wkhtmltopdf: /opt/lampp/lib/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by /usr/local/bin/wkhtmltopdf)
  | /usr/local/bin/wkhtmltopdf: /opt/lampp/lib/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /usr/local/bin/wkhtmltopdf)
  | /usr/local/bin/wkhtmltopdf: /opt/lampp/lib/libz.so.1: version `ZLIB_1.2.9' not found (required by /usr/lib/x86_64-linux-gnu/libpng16.so.16)

`

**Comment by mikehaertl** [comment]
Hmm, sorry, I have no idea. If you find out what's wrong and find that we can fix/improve something here, let us know.
